lib/connection/process.coffee

```diff
@@ -11,28 +32,51 @@ module.exports =
       process.env.HOME || process.env.USERPROFILE
 
   jlpath: -> atom.config.get("julia-client.juliaPath")
-  # TODO: this is very naïve.
-  jlargs: -> atom.config.get("julia-client.juliaArguments").split ' '
+
+  checkExe: (path, cb) ->
+    which = if process.platform is 'win32' then 'where' else 'which'
```

Comment:
We can just add an early return if `fs.fileExists(path)`. That will be far faster than shelling out on Windows anyway.